### DISCUSSION TITLE
Help with Authelia -- jwks

### DISCUSSION THREAD

**Discussion by nothing2obvi** [9p4/jellyfin-plugin-sso#181] [Q&A]
**Describe the bug**
Please lmk if this is the wrong place to ask this, but I can't get Authelia up and running with the provided details from the documentation because it's telling me the `jwks` option needs to be filled out. What do I put there?

**To Reproduce**

1. Use documentation to fill out details for Authelia configuration
2. Start Authelia container
3. Get error about jwks

**Expected behavior**
Authelia should be running

**Screenshots**

```
04/14/2024
11:58:59 PM
files=
[/config/configuration.yml]
filters=
[]
level=
debug
msg=
Loaded Configuration Sources
time=
2024-04-15T06:58:59Z
04/14/2024
11:58:59 PM
fields.level=
debug
file=
format=
keep_stdout=
false
level=
debug
msg=
Logging Initialized
time=
2024-04-15T06:58:59Z
04/14/2024
11:58:59 PM
gid=
0
gids=
1,2,3,4,6,10,11,20,26,27
level=
debug
msg=
Process user information
name=
root
time=
2024-04-15T06:58:59Z
uid=
0
username=
root
04/14/2024
11:58:59 PM
level=
error
msg=
Configuration: identity_providers: oidc: option `jwks` is required
stack=
github.com/authelia/authelia/v4/internal/commands/context.go:324 (*CmdCtx).ConfigValidateLogRunE github.com/authelia/authelia/v4/internal/commands/context.go:186 NewRootCmd.(*CmdCtx).ChainRunE.func1 github.com/spf13/cobra@v1.8.0/command.go:968 (*Command).execute github.com/spf13/cobra@v1.8.0/command.go:1115 (*Command).ExecuteC github.com/spf13/cobra@v1.8.0/command.go:1039 (*Command).Execute github.com/authelia/authelia/v4/cmd/authelia/main.go:10 main runtime/internal/atomic/types.go:194 (*Uint32).Load runtime/asm_arm64.s:1222 goexit
time=
2024-04-15T06:58:59Z
04/14/2024
11:58:59 PM
level=
fatal
msg=
Can't continue due to the errors loading the configuration
stack=
github.com/authelia/authelia/v4/internal/commands/context.go:327 (*CmdCtx).ConfigValidateLogRunE github.com/authelia/authelia/v4/internal/commands/context.go:186 NewRootCmd.(*CmdCtx).ChainRunE.func1 github.com/spf13/cobra@v1.8.0/command.go:968 (*Command).execute github.com/spf13/cobra@v1.8.0/command.go:1115 (*Command).ExecuteC github.com/spf13/cobra@v1.8.0/command.go:1039 (*Command).Execute github.com/authelia/authelia/v4/cmd/authelia/main.go:10 main runtime/internal/atomic/types.go:194 (*Uint32).Load runtime/asm_arm64.s:1222 goexit
time=
2024-04-15T06:58:59Z
```

**Configuration**
N/A

**Versions (please complete the following information):**

- OS: [Docker Desktop for Mac 4.29.0]
- Browser: [Opera]
- Jellyfin Version: [10.8.13]
- Plugin Version: [3.5.2.3]

**Comment by 9p4** [comment]
https://deploy-preview-7148--authelia-staging.netlify.app/integration/openid-connect/jellyfin/

**Comment by nothing2obvi** [comment]
@9p4

Thanks for your reply. I followed that guide but get completely lost because of the error I get in my original post about the `jwks` section. I'm not sure if I'm supposed to set up an OIDC provider section in Authelia, with a `jwks` section, and a corresponding `jwks` section for Jellyfin in Authelia.

**Comment by 9p4** [comment]
Authelia needs the jwks section, not the plugin.

**Comment by nothing2obvi** [comment]
I think I got it. The documentation was just a bit confusing for me. Thanks!
